cypress/e2e/Login.cy.js

```diff
@@ -0,0 +1,72 @@
+///<reference types="cypress" /> 
+
+import Login from '../POMs/LoginPage'
+
+const Li = new Login()
+
+describe('Swag Labs Login Scenarios', () => {
+
+  beforeEach( function () {
+
+    cy.visit("https://www.saucedemo.com/")
```

Comment:
We can move this in the cypress.config.js file as BaseUrl

Reply:
Got it. I will do that.